dtfs-central-api/src/services/repositories/utilisation-reports-repo/utilisation-reports-repo.ts

```diff
@@ -50,36 +50,22 @@ export const getOpenReportsBeforeReportPeriodForBankId = async (reportPeriodStar
   const collection = await db.getCollection(DB_COLLECTIONS.UTILISATION_REPORTS);
 
   return await collection
```

Comment:
Summary of changes:
* correct match on `bankId` to `'bank.id'`
* replace `aggregate` with `find` (in a previous version I had a multi-step query, but didn't need it in the end)